app/vmselect/promql/exec_test.go

```diff
@@ -8071,7 +8071,7 @@ func TestExecSuccess(t *testing.T) {
 		}}
 		r3 := netstorage.Result{
 			MetricName: metricNameExpected,
-			Values:     []float64{4, 3, 2, 1, 0, -1},
```

Comment:
Bump `2000` in test expr to `2200`, since now `removeCounterResets` wouldn't return negative value,